### PR TITLE
feat: Repo recommendations UI

### LINKED ISSUE #1000: Implement Repo Recommendations in the Design System
Screen Shot 2023-03-13 at 5 53 51 PM

[[Figma: Repo Recommendations](https://www.figma.com/file/768ifqOGWrsRN29Igkc4ip/User-Flows?node-id=139%3A537&t=B0fmUuiTTS4HUVTm-1)]

### PR DESCRIPTION
**Pull request by Deadreyo**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

- Added `TooltipProvider` into `hover-card` to not require adding tooltip provider everywhere that component is used.
- Added missing story to stacked avatar
- Added Repo Recommendation Card to the design system.

## Related Tickets & Documents

fixes #1000 
### Repo Recommendations Card:
firefox_DWfrYmIQko
firefox_p79RXSZ2Ak

## Added tests?

- [x] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [x] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by Deadreyo** [comment]
Currently all tab "pages" (i.e. highlight & contributions sections) in `/username` are coded directly into the `contributor-profile-tab.ts`. This can use a refractor to separate each section into its own component to have the code easier to handle and add upon. I will start this by having this recommendations tab as a separate UI component

cc @OgDev-01 @brandonroberts

**Review by bdougie — APPROVED** [review]

File: components/molecules/RecommendedRepoCard/recommended-repo-card.tsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+}
+
+const RecommendedRepoCard = ({ fullname, description, issues, pullRequests, stars, contributions }: RecommendedRepoCardProps): JSX.Element => {
+
+  const [owner, name] = fullname.split("/");
+
+  return (
+    <div className="p-4 border rounded-2xl bg-white w-full min-w-fit space-y-1 relative">
+      <div className="flex justify-between w-full">
+        <div className="flex space-x-1 items-center">
```
Comment (OgDev-01) [inline components/molecules/RecommendedRepoCard/recommended-repo-card.tsx:24]: use `gap` if you have flex defined

**Review by OgDev-01 — APPROVED** [review]
🍕

File: components/molecules/HoverCardWrapper/hover-card-wrapper.tsx [hunk 2]
```diff
@@ -5,6 +5,7 @@ import { useSingleContributor } from "lib/hooks/useSingleContributor";
 import ContributorHoverCard, { ContributorsProfileType } from "../ContributorHoverCard/contributor-hover-card";
 
 import roundedImage from "lib/utils/roundedImages";
+import { TooltipProvider } from "@radix-ui/react-tooltip";
```
Comment (brandonroberts) [inline components/molecules/HoverCardWrapper/hover-card-wrapper.tsx:8]: We already have a `TooltipProvider` in the root of the app, if you need this for the Storybook story to work, wrap the component in the story itself.

**Review by brandonroberts — CHANGES_REQUESTED** [review]

File: components/molecules/HoverCardWrapper/hover-card-wrapper.tsx (hunk 2 above)
Comment (brandonroberts) [reply components/molecules/HoverCardWrapper/hover-card-wrapper.tsx]: We can revisit this if we need to customize the behavior at a more granular level

**Pull request merged**